Documentation/Unio0_9_0MigrationGuide.md

```diff
@@ -0,0 +1,17 @@
+# Unio 0.9.0 Migration Guide
+
+Unio 0.9.0 introduces some breaking changes.
+
+Those changes are related [SR-12081](https://bugs.swift.org/browse/SR-12081).
+
+## Classes
+
+- [ADD] `AnyLogicBasedStream<Input: InputType, Output: OutputType>`
+
+## Typealias
+
+- [REPLACE] `typealias UnioStream<Logic: LogicType> = PrimitiveStream<Logic> & LogicType` -> `typealias UnioStream<Logic: LogicType> = AnyLogicBasedStream<Logic.Input, Logic.Output> & LogicType`
+
+## Note
+
+If you want to use existing UnioStream initializer (less than 0.8.0), please define `typealise UnioSteram<Logic: LogicType> = SR12081.UnioSteram<Logic>` in your Application Target.
```

Comment:
It'd be great if the real "migration" way to use the new UnioStream, which is to use `convenience init` and `self.init`, stated anywhere in this document. This looks too CHANGELOG-ish for now I think?

Reply:
I've added some comments 442bdea357bc8f298e5cf95de60102053882f48e .